src/debugger/Debugger.js

```diff
@@ -229,29 +236,37 @@ export class DebugServer {
     let context = this._realm.contextStack[stackIndex];
```

Comment:
You can use "this._realm.getRunningContext()" instead.

Reply:
The frameId passed in from the UI can correspond to any of the contexts in the stack.